src/ui/webeiddialog.cpp

```diff
@@ -231,57 +240,53 @@ void WebEidDialog::onMultipleCertificatesReady(
 void WebEidDialog::onSingleCertificateReady(const QUrl& origin,
                                             const CardCertificateAndPinInfo& certAndPin)
 {
-    try {
```

Comment:
The general rule in Qt is that exceptions should never be thrown from slots, so it is a good practice to wrap the slot method body in `try/catch`. What do you think?

Reply:
Can you point where it does throw?

Reply:
You are right that currently none of the methods throw. But not wrapping them in `try/catch` makes the slot method fragile for future changes and moves us away from exception safety and using exceptions by default. I would keep the `try/catch` as a safety net that encourages exception usage.